configuration/items.md

```diff
@@ -206,7 +206,21 @@ Number Livingroom_Temperature "Temperature [%.1f °C]"
 The state of an Item depends on the Item type, the Channel bound to the Item, and internal or external events.
 A analogy can be drawn between the state of an Item and the value of a variable in a computer program.
 
-<!-- TODO add a complete description about the actual state of an Item, e.g. Initialization, UNDEF, Binding etc. -->
+{: #item-state}
+#### Item State
+
+This section provides information about what a user can expect regarding the behavior of the state of an Item.
+
+-   Items are created with a state of `NULL`.
+
+-   Operations in openHAB such as a user interacting with the Item using the `Basic UI`, or a Binding updating the state of an Item will change the state of the Item
```

Comment:
backticks should be used for code-related text only. Adding some around "Basic UI" was by mistake?

Reply:
Agree - will remove backticks